lib/ansible/executor/task_executor.py

```diff
@@ -80,6 +80,31 @@ def __init__(self, host, task, job_vars, play_context, new_stdin, loader, shared
         self._final_q = final_q
         self._loop_eval_error = None
 
+        from collections import defaultdict
+        from ansible.module_utils._text import to_text, to_bytes, to_native
+        from yaml import load as yaml_load
```

Comment:
TODO: move this reading up to the play-context level so that it happens once per play, rather than once per task (which is clearly too frequent).